tools/SetupFlow/DevHome.SetupFlow/Controls/SetupShell.xaml.cs

```diff
@@ -57,4 +57,10 @@ public SetupShell()
     public static readonly DependencyProperty SetupShellContentProperty = DependencyProperty.RegisterAttached(nameof(SetupShellContent), typeof(object), typeof(SetupShell), new PropertyMetadata(null));
     public static readonly DependencyProperty HeaderTemplateProperty = DependencyProperty.RegisterAttached(nameof(HeaderTemplate), typeof(ControlTemplate), typeof(SetupShell), new PropertyMetadata(null));
     public static readonly DependencyProperty OrchestratorProperty = DependencyProperty.RegisterAttached(nameof(Orchestrator), typeof(SetupFlowOrchestrator), typeof(SetupShell), new PropertyMetadata(null));
+
+    private void OnLoaded(object sender, RoutedEventArgs e)
+    {
+        // Focus on the first focusable element inside the shell content
+        ShellContent.Focus(FocusState.Programmatic);

```

Comment:
If I recall, I think the dashboard page uses a focus manager object? might be interesting (not in this PR) to see if we can leverage that for the whole app.

Reply:
Great, thanks for the suggestion! I didn't find it in the dashboard but I found an example online 😊 Updated